x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/set.tsx

```diff
@@ -57,7 +52,6 @@ const fieldsConfig: FieldsConfig = {
       />
     ),
   },
-  /* Optional fields config */
```

Comment:
I thought of removing it since they were all optional now, but since you mention Ill add it back to this config and will specify the other config as required